backend/FwLite/FwLiteProjectSync.Tests/Fixtures/Sena3SyncFixture.cs

```diff
@@ -0,0 +1,119 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using MiniLcm;
+using SIL.IO;
+using SIL.Progress;
+
+namespace FwLiteProjectSync.Tests.Fixtures;
+
+public class Sena3Fixture : IAsyncLifetime
+{
+    private readonly AsyncServiceScope _services;
+
+    public CrdtFwdataProjectSyncService SyncService =>
+        _services.ServiceProvider.GetRequiredService<CrdtFwdataProjectSyncService>();
+
+    public IServiceProvider Services => _services.ServiceProvider;
+    private IDisposable _cleanup;
+    private readonly LexboxConfig lexboxConfig;
+    private readonly HttpClient http;
+    public CrdtMiniLcmApi CrdtApi { get; set; } = null!;
+    public FwDataMiniLcmApi FwDataApi { get; set; } = null!;
+    private bool AlreadyLoggedIn { get; set; } = false;
+
+    public Sena3Fixture()
+    {
+        var services = new ServiceCollection()
+            .AddSyncServices(nameof(Sena3Fixture), false);
+        services.AddOptions<LexboxConfig>()
+            .BindConfiguration("LexboxConfig")
+            .Configure(c =>
+            {
+                // TODO: How do I set default values if and only if they're not already set (e.g., via environment variables)?
+                c.LexboxUrl = "http://localhost/";
+                c.LexboxUsername = "admin";
+                c.LexboxPassword = "pass";
+            })
+            .ValidateDataAnnotations()
+            .ValidateOnStart();
+        var rootServiceProvider = services.BuildServiceProvider();
+        _cleanup = Defer.Action(() => rootServiceProvider.Dispose());
+        _services = rootServiceProvider.CreateAsyncScope();
+        lexboxConfig = Services.GetRequiredService<IOptions<LexboxConfig>>().Value;
+        var factory = Services.GetRequiredService<IHttpClientFactory>();
+        http = factory.CreateClient(nameof(Sena3Fixture));
+    }
+
+    public async Task InitializeAsync()
+    {
+        var sena3MasterCopy = await DownloadSena3();
+        var projectName = "sena-3_" + Guid.NewGuid().ToString("N");
+
+        var projectsFolder = Services.GetRequiredService<IOptions<FwDataBridgeConfig>>().Value
+            .ProjectsFolder;
+        if (Path.Exists(projectsFolder)) Directory.Delete(projectsFolder, true);
+        Directory.CreateDirectory(projectsFolder);
+        var fwDataProject = new FwDataProject(projectName, projectsFolder);
+        var fwDataProjectPath = Path.Combine(fwDataProject.ProjectsPath, fwDataProject.Name);
+        DirectoryHelper.Copy(sena3MasterCopy, fwDataProjectPath);
+        File.Move(Path.Combine(fwDataProjectPath, "sena-3.fwdata"), fwDataProject.FilePath);
+
+        FwDataApi = Services.GetRequiredService<FwDataFactory>().GetFwDataMiniLcmApi(fwDataProject, false);
+
+        var crdtProjectsFolder =
+            Services.GetRequiredService<IOptions<LcmCrdtConfig>>().Value.ProjectPath;
+        if (Path.Exists(crdtProjectsFolder)) Directory.Delete(crdtProjectsFolder, true);
+        Directory.CreateDirectory(crdtProjectsFolder);
```

Comment:
@hahn-kev - Isn't this code block already deleting any old projects at the start of the test run? I think I might be misunderstanding what you're asking for in https://github.com/sillsdev/languageforge-lexbox/pull/1256#pullrequestreview-2460937345 because it seems to me that that's already being done. (And lines 63-66 do the same for the .fwdata project). What part of the code is failing to clean up old projects?

Reply:
yeah you're right they are getting cleaned up, maybe I had broken that at some point while I was testing it and didn't realize the projects were being cleaned up by the end